### DISCUSSION TITLE
v3.6.7

### DISCUSSION THREAD

**Discussion by wwkimball** [wwkimball/yamlpath#188] [General]
Bug Fixes:
* Release 3.6.6 had a broken package; it was unexpectedly missing the required
  ruamel.yaml patch found in yamlpath.patches.timestamp.  Thanks to
  https://github.com/tsinggggg and https://github.com/kaniblu for reporting it!

<hr /><em>This discussion was created from the release <a href='https://github.com/wwkimball/yamlpath/releases/tag/v3.6.7'>v3.6.7</a>.</em>

**Comment by Namphibian** [comment]
Awesome. Spent the last 4 hours pulling out my hair to get it working and was about to file a bug report. Massive thanks! Using Ubuntu 22.04 and Python 3.9 and it all good.

**Reply by wwkimball** [reply]
I'm really sorry for the frustration.  I was just as surprised as everyone else that the package was broken!

The GitHub Action for publishing release packages runs without any post-build package testing, as far as I can tell.  It'd be awesome if it would run all of my tests against the _built_ package in a virtual environment to detect packaging issues before publishing.

**Reply by Namphibian** [reply]
Its all good. Learnt a lot more about PIP and other part of Python so good exposure. I have a few ideas I want to discuss with you. Some potential small enhancements to the library that could be useful. Awesome work mate really love this library.
